cmd/cli/main.go

```diff
@@ -43,6 +48,16 @@ var rootCmd = &cobra.Command{
 	Long: commandName + " is the command line tool for managing the auditing resource repository",
 }
 
+var getCmd = &cobra.Command{
+	Use:   "get [-a author] [-s since] [-u until] [-r resource] [-n namespace] [-f name]",
+	Short: "get changes by author, time range, and filepath",
+	Run:   runGet,
+	Example: ` Getting changes by author and filepath
+    $ auditctl get -a kubernetes-admin -r k8s-policies -n default -f allow-client1.yaml
+    [{"sha":"a75dc67fd950b5ed052897b981c8d7b2cb05e9a5","author":"kubernetes-admin","Message":"Deleted K8s network policy default/allow-client1"}]
```

Comment:
```suggestion
    [{"sha":"a75dc67fd950b5ed052897b981c8d7b2cb05e9a5","author":"kubernetes-admin","message":"Deleted K8s network policy default/allow-client1"}]
```

Reply:
This is just the way that the commit object is returned, which is what I print.

Reply:
this should no longer be the case after fixing the `Change` struct:

```
type Change struct {
	Sha     string `json:"sha"`
	Author  string `json:"author"`
	Message string `json:"message"`
}
```

The `json` tag determines the field name here. You may want to try again.